ttnn/cpp/ttnn/operations/data_movement/concat/concat.hpp

```diff
@@ -7,7 +7,7 @@
 #include "ttnn/tensor/types.hpp"
 #include "ttnn/operations/core/core.hpp"
 
-#include "ttnn/deprecated/tt_dnn/op_library/concat/concat_op.hpp"
+#include "ttnn/cpp/ttnn/operations/data_movement/concat/device/concat_device_operation.hpp"
```

Comment:
I think you really only need "ttnn/decorators.hpp"

Reply:
Done